webrtc.html

```diff
@@ -2363,11 +2363,14 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
+                <p>As noted in <span data-jsep="ice-candidate-pool">[[!JSEP]]</span>,
```

Comment:
Unfortunately section 5.8 doesn't mention about triggering ICE gathering. The only place it is hinted that ICE gathering is triggered by setLocalDescription in section 3.5.4 for ICE candidate pool:

> When setLocalDescription is eventually called, and the JSEP implementation goes to gather the needed ICE candidates, it SHOULD start by checking if any candidates are available in the pool.